_posts/2015-10-31-setting-up-babel-6.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: post
+title:  "Setting up Babel 6"
+author: James Kyle
+date:   2015-10-31 17:30:00
+categories: announcements
+share_text: "Setting up Babel 6"
+---
+
+Babel 6 is fresh off the press and we’re still getting a lot sorted out. In the past two days has seen more activity on GitHub and Slack than ever before. Everyone who has been helping out has been doing a great job.
```

Comment:
Don't need the `In` in `In the past two days` or maybe change to `In the past two days, we've seen more..`